config/default/media.type.document.yml

```diff
@@ -20,7 +20,7 @@ third_party_settings:
     unpublish_revision: false
 id: document
 label: Document
-description: 'A Document file is written information.'
```

Comment:
Please check whether we need these changes.

Reply:
reverted